src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -350,8 +350,13 @@ public enum SandboxType {
     
     private int memoryLimitMB = DEFAULT_MEMORY_LIMIT_MB;
     private SandboxType sandboxType = DEFAULT_SANDBOX_TYPE;
-    private String sandboxCmdLine = Constants.PC2_INTERNAL_SANDBOX_COMMAND_LINE;
-    private String sandboxProgramName = Constants.PC2_INTERNAL_SANDBOX_PROGRAM_NAME;
+    private String sandboxCmdLine = null;
+    private String sandboxProgramName = null;
```

Comment:
I'm not sure I understand the logic of deleting the default assignments for a Problem's `sandboxCmdLine` and `sandboxProgramName`.  Every Problem has a default memory limit (for example); shouldn't it also have a default sandbox command line/program name?  (Is it because the default sandbox TYPE is `Sandbox_Type.NONE`?)

Reply:
Yes.  This logic has been changed.  If the value(s) is(are) null, it is dealt with in updateSandboxInfo().
There's a comment there that explains it.